state-chain/primitives/src/lib.rs

```diff
@@ -58,7 +58,7 @@ pub const FLIPPERINOS_PER_FLIP: FlipBalance = 10u128.pow(FLIP_DECIMALS);
 pub const DEFAULT_FEE_SATS_PER_KILO_BYTE: u64 = 102400;
 
 // Approximate values calculated
-pub const INPUT_UTXO_SIZE_IN_BYTES: u64 = 75;
+pub const INPUT_UTXO_SIZE_IN_BYTES: u64 = 78;
```

Comment:
worth adding a comment explaining why it's this value

Reply:
done